Source/Shaders/PointPrimitiveCollectionVS.glsl

```diff
@@ -0,0 +1,171 @@
+uniform float u_maxTotalPointSize;
+
+attribute vec4 positionHighAndSize;
+attribute vec4 positionLowAndOutline;
+attribute vec4 compressedAttribute0;        // color, outlineColor, pick color
+attribute vec4 compressedAttribute1;        // show, translucency by distance, some free space
+attribute vec4 scaleByDistance;             // near, nearScale, far, farScale
+
+varying vec4 v_color;
+varying vec4 v_outlineColor;
+varying float v_innerPercent;
+varying float v_pixelDistance;
+
+#ifdef RENDER_FOR_PICK
+varying vec4 v_pickColor;
+#endif
+
+float getNearFarScalar(vec4 nearFarScalar, float cameraDistSq)
```

Comment:
Could you make this a `czm_` function that is shared with billboards and has a unit test?